pants

```diff
@@ -20,6 +20,14 @@ PANTS_BOOTSTRAP="${PANTS_HOME}/bootstrap-$(uname -s)-$(uname -m)"
 
 VENV_VERSION=15.0.2
 
+# This requirement is installed alongside pantsbuild.pants to hack around
+# interpreters that have newer setuptools already installed, effectively
+# re-installing an older setuptools and pinning low to the version range
+# known to be ingestable by pants and pex. See:
+#   https://github.com/pantsbuild/pants/issues/3948
+#   https://github.com/pantsbuild/setup/issues/14
+SETUPTOOLS_REQUIREMENT="setuptools>=2.2,<20.11"
```

Comment:
should the lower range of this pin be `>=5.4.1` given that's whats in our requirements.txt?